### ISSUE TITLE
Ewey GUI (hah)

### ISSUE THREAD

**Issue by subrosaassociates** [codewatchorg/sqlipy#19]
Hey man, great job! 

I have a small issue

<img width="1142" alt="screen shot 2018-01-09 at 00 12 04 copy" src="https://user-images.githubusercontent.com/34861344/34698793-f37a5046-f4d1-11e7-89bc-22d934185f35.png">

St [op]/St [art] ? ! :)

**Comment by codewatchorg** [comment]
Yeah, I am not much of a developer and even less of a Java dev.  I am using Java GridBagLayout, which is very limited, but at the time it was the easiest for me to get up and running.  If you have a high-res display, are using large fonts, or a few other instances, then some of the text won't all display correctly.  I need to change the layout, but I am limited on time and still pretty limited in my knowledge of Java layouts.

**Comment by codewatchorg** [comment]
It is "Stop" by the way.  The only way to start anything is either in the API start/stop tab, or the scan start/stop tab.

**Comment by subrosaassociates** [comment]
Ooooh, St is.stop

Thanks man, that'll do me :)

Its a great interface well done!

**Comment by fuuddanni** [comment]
Hi, i´m also on an highres monitor, is there a way to change the UI in Burp (32px Layout): Thx. 
![sqlmap_burp_gui](https://user-images.githubusercontent.com/6312269/46922012-d82add80-d002-11e8-8ddd-b6177e51d2fd.png)

**Comment by codewatchorg** [comment]
I feel your pain, but I haven't had time to update the layout to work for low and hi-res monitors (I have a high res monitor and just change the font size within Burp to make it more manageable - but it is still a pain).
